src/nccl_ofi_scheduler.c

```diff
@@ -38,14 +72,14 @@ static inline int set_schedule_by_threshold(nccl_net_ofi_threshold_scheduler_t *
 					    nccl_net_ofi_schedule_t *schedule)
 {
 	int ret = 0;
+	int num_stripes = 0;
 
-	/* Number of stripes is atleast 1 for zero-sized messages and at most equal to num of rails */
-	int num_stripes =
-		(int)NCCL_OFI_MAX(1, NCCL_OFI_MIN(NCCL_OFI_DIV_CEIL(size, scheduler->min_stripe_size), (unsigned)num_rails));
-	if (OFI_UNLIKELY(num_rails == 0)) {
+	if (OFI_UNLIKELY(num_rails <= 0)) {
```

Comment:
I missed in the previous commit that this was checking `num_rails` and not `num_stripes`.  This is a silly check and should just be an assert.

Reply:
Done.